types/index.ts

```diff
@@ -138,12 +87,40 @@ export interface EngineCondition<T extends AudienceDefinitionFilter> {
   rules: EngineConditionRule[];
 }
 
-export interface PingResponse {
-  gdprApplies?: boolean;
+// Audience cache interfaces
+
+export interface MatchedAudience {
+  id: string;
+  matchedAt: number;
+  expiresAt: number;
+  matchedOnCurrentPageView: boolean;
+}
+
+export interface CachedAudienceMetaData {
+  cachedAt: number;
+  audiences: AudienceMetaData[];
 }
 
+export interface AudienceMetaData {
+  id: string;
+  version: number;
+}
+
+// Storage interfaces
+
+export enum StorageKeys {
+  PAGE_VIEWS = 'edkt_page_views',
+  MATCHED_AUDIENCES = 'edkt_matched_audiences',
+  MATCHED_AUDIENCE_IDS = 'edkt_matched_audience_ids',
+  CACHED_AUDIENCES = 'edkt_cached_audiences',
+  CACHED_AUDIENCE_META_DATA = 'edkt_cached_audience_meta_data',
+}
+
+// Network interfaces
+
 export interface TCData {
-  gdprApplies?: boolean;
+  // TODO I've put this back since the tests strongly suggests this should be considered
```

Comment:
Ok great it is needed so no TODO?

Reply:
The value is not being used at all.
It is present on tests and used there as if it were important, but the actual method holds no reference to it.
The `TODO` is to check why it exists and which check is missing on the method.
I should have expressed it clearer, tho.

Reply:
It is needed and is an external interface https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#tcdata

We can add link - and remove this todo I think?

Reply:
That explains! haha
Done!